plugin/trino-clickhouse/src/test/java/io/trino/plugin/clickhouse/BaseClickHouseTypeMapping.java

```diff
@@ -442,7 +442,7 @@ public void testDecimal()
     @Test
     public void testClickHouseChar()
     {
-        // ClickHouse char is FixedString, which is arbitrary bytes
+        // ClickHouse char is String, which is arbitrary bytes
```

Comment:
This change is unrelated to "Enable varchar equality predicate pushdown for ClickHouse-connector". Please separate a commit.

Reply:
Moved to the unit test commit.